docs/installation/post-install.rst

```diff
@@ -84,6 +84,37 @@ For more details on the available options for storage management, see
    - Sanity check
    - Troubleshooting if needed
 
+Loki volume sizing
+""""""""""""""""""
+
+Since the storage needs for logs greatly depends on the workload and the
+type of application that run on top of the MetalK8s cluster, you need to
+refer to the documentation provided by your applications to define the ideal
+size for the volume.
+
+We still provide some hints for the worst case, which is very unlikely.
+If your logs are gibberish, which makes them almost incompressible,
```

Comment:
`a random stream of bytes` may be better than `gibberish`

Reply:
```suggestion
If the entropy of log messages is high, which makes them almost incompressible,
```

;)

Reply:
You're right, fixed.

Reply:
@NicolasT Oh yes, even better, because it's not only for random stuff, but also when there are only very different log messages.